### ISSUE TITLE
bug: admonition class names don't remove spaces (e.g. "See Also")

### ISSUE THREAD

**Issue by Pugio** [executablebooks/markdown-it-docutils#22]
### Describe the bug

**context**
```
   ```{seealso}
      seealso
   `` `
```

**expectation**
```html
<aside class="admonition seealso">
  <header class="admonition-title">See Also</header>
  <p>seealso</p>
</aside>
```

**bug**
```html
<aside class="admonition see also">
  <header class="admonition-title">See Also</header>
  <p>seealso</p>
</aside>
```

**solution**
[admonitions.ts#L31](https://github.com/executablebooks/markdown-it-docutils/blob/56bd6d1929b6e1120dbbdfa7a52ba759bdc2034c/src/directives/admonitions.ts#L31) should read:

```typescript
if (this.title) adToken.attrJoin("class", this.title.replace(/ /g, '').toLowerCase())
```

Personally, I would prefer it to read `this.title.replace(/ +/g, '-')` – converting admonition title spaces into kebab-case, as this would make the resulting CSS styling nicer to read. Doing so would mean updating the current CSS rules for `seealso` however, and so might be a larger change.

### Reproduce the bug

```
   ```{seealso}
      seealso
   `` `
```

**Comment by chrisjsewell** [comment]
Yep thanks, I would note that sphinx adds it as `seealso`: https://github.com/sphinx-doc/sphinx/blob/e54b63d3eead95ef3fac1906deae54b6972573e4/sphinx/writers/html.py#L289, so I would want to stick to that

**Comment by rowanc1** [comment]
This is fixed in #22! Thanks for the report @Pugio.

**Comment by rowanc1** [comment]
This has been merged into main, but not yet released. Hoping that we can get a release out soon!

**Comment by rowanc1** [comment]
Released in 0.1.2:

https://github.com/executablebooks/markdown-it-docutils/releases/tag/v0.1.2
